src/elarm_alarmlist.erl

```diff
@@ -151,36 +151,36 @@ al_test_() ->
      ]}.
 
 just_started() ->
-    {ok,State} = init([]),
-    ?assertEqual({{ok,[]}, State}, get_alarms(State)),
-    EvtId = erlang:now(),
-    ?assertEqual({{error, not_active},State}, get_alarm(EvtId, State)),
-    ?assertEqual({{error, not_active},State}, get_alarm(full, disk_1, State)).
+    {ok, State} = init([]),
+    ?assertEqual({{ok, []}, State}, get_alarms(State)),
+    EvtId = erlang:timestamp(),
```

Comment:
We cannot use erlang:timestamp because that was introduced only in Erlang /OTP18, we need to support versions >= R15B03.